src/test/java/io/fabric8/maven/docker/service/RunServiceTest.java

```diff
@@ -277,6 +279,38 @@ public void testWithException() throws Exception {
         runService.stopContainer(container, createImageConfig(SHUTDOWN_WAIT, 0), false, false);
     }
 
+    @Test
+    public void testWithMultipleStopExceptions() throws Exception {
+        GavLabel testLabel = new GavLabel("Im:A:Test");
+
+        String firstName = "first-container:latest";
+        ImageConfiguration first = new ImageConfiguration();
+        first.setName(firstName);
+
+        String secondName = "second-container:latest";
+        ImageConfiguration second = new ImageConfiguration();
+        second.setName(secondName);
+
+        tracker.registerContainer(firstName, first, testLabel);
+        tracker.registerContainer(secondName, second, testLabel);
+
+        LogOutputSpecFactory logOutputSpecFactory = new LogOutputSpecFactory(true, true, null);
+
+        new Expectations(){{
+           docker.stopContainer(firstName, 0); result = new DockerAccessException("TEST one");
+           docker.stopContainer(secondName, 0); result = new DockerAccessException("TEST two");
+        }};
+
+        runService = new RunService(docker, queryService, tracker, logOutputSpecFactory, log);
+
+        try {
```

Comment:
I think maybe we can improve this by using `assertThrows` method like this:
```java
Exception thrownException = assertThrows(DockerAccessException.class, () -> runService.stopStartedContainers(false, true, true, testLabel));
assertEquals("(TEST two,TEST one)", thrownException.getLocalizedMessage());
```
WDYT?